File: Plugins/Aspose_Tasks_Java_for_Ruby/lib/asposetasksjava/Projects/createemptyproject.rb
module Asposetasksjava
  module CreateEmptyProject
    def initialize()
        data_dir = File.dirname(File.dirname(File.dirname(File.dirname(__FILE__)))) + '/data/'
                
        # Instantiate project object
        project = Rjb::import('com.aspose.tasks.Project').new

        project_stream =  Rjb::import('java.io.FileOutputStream').new(data_dir + "Project1.xml")
        project.save(project_stream, Rjb::import('com.aspose.tasks.SaveFileFormat').XML)
        project_stream.close()

        # Display Status
        puts "Created project Successfully."
    end
  end
end

File: Plugins/Aspose_Tasks_Java_for_PHP/src/aspose/tasks/WorkingWithProjects/CreateEmptyProject.php
<?php
namespace Aspose\Tasks\WorkingWithProjects;

use com\aspose\tasks\Project as Project;
use com\aspose\tasks\SaveFileFormat as SaveFileFormat;

use java\io\FileOutputStream as FileOutputStream;

class CreateEmptyProject
{
    public static function run($dataDir=null)
    {
        # Instantiate project object
        $project = new Project();

        $project_stream =  new FileOutputStream($dataDir . "Project1.xml");
        $saveFileFormat=new SaveFileFormat();
        $project->save($project_stream, $saveFileFormat->XML);
        $project_stream->close();

        # Display Status
        print "Created project Successfully.".PHP_EOL;

    }
}
?>